recipes/server_os_packages.rb

```diff
@@ -58,6 +58,4 @@
 end
 
 # dependencies
-os_packages.each do |p|
-  package p
-end
+package os_packages unless os_packages.empty?
```

Comment:
No need to check if os_packages is empty. It will filled always (by the above code).
Also, same code was introduced by me in #292 ;-)